src/troute-network/troute/AbstractRouting.py

```diff
@@ -108,7 +108,7 @@ def unrefactored_topobathy_df(self):
 class MCOnly(AbstractRouting):
 
     def __init__(self, hybrid_params):
-        self.hybrid_params = hybrid_params
+        self.hybrid_params = None
```

Comment:
should probably drop `hybrid_params` from the arg list here as well...

Reply:
Replacing `hybrid_params` with `_` here.